cpp/include/cuml/fil/fil.h

```diff
@@ -187,6 +188,14 @@ void init_sparse(const cumlHandle& h, forest_t* pf, const int* trees,
 void from_treelite(const cumlHandle& handle, forest_t* pforest,
                    ModelHandle model, const treelite_params_t* tl_params);
 
+
+void from_multi_treelites(const cumlHandle& handle, forest_t* pforest,
+                          ModelHandle model, ModelHandle model_2,
+                          std::vector<ModelHandle*> model_vector,
+                          const treelite_params_t* tl_params);
+
+ModelHandle* concatenate_trees(const cumlHandle& handle,
+                               const std::vector<ModelHandle *> treelite_handles);
```

Comment:
`ModelHandle` is already a pointer, and it is a C type, so no destructor. Therefore, it is best to pass `treelite_handles` as `const std::vector<ModelHandle>&`.